src/model/scripts/bpe_training/train.py

```diff
@@ -0,0 +1,77 @@
+from datasets import load_dataset
+from tokenizers import ByteLevelBPETokenizer
+from nlpo3 import segment, load_dict
+import argparse
+import os
+
+SPECIAL_TOKENS = [
```

Comment:
I think you should create `special_tokens.json` file to save these token and load from file in args.

Reply:
done